### ISSUE TITLE
fvtt-CommunityLighting ignores foundry routePrefix for assets paths

### ISSUE THREAD

**Issue by js-murph** [BlitzKraig/fvtt-CommunityLighting#21]
fvtt-CommunityLighting seems to ignore the routePrefix foundry option when configured for determining its default asset paths.

For example if routePrefix = "thing" all content is going to be at the base path of https://mydomain/thing/. The current behaviour of fvtt-CommunityLighting results in the following...

```
GET https://mydomain/modules/CommunityLighting/lights.json 404
```

It should however be doing...

```
GET https://mydomain/thing/modules/CommunityLighting/lights.json
```

**Comment by BlitzKraig** [comment]
Thank you for the report, I'll get this in as a high priority fix tonight

**Comment by BlitzKraig** [comment]
Hi js-murph,
This issue should be resolved in 0.3.1, just released.

Thanks for the feedback!
